src/app.scss

```diff
@@ -10,15 +10,20 @@
 
 html {
 	margin: 0;
-	padding: 0;
+	overflow-y: auto;
+	overflow-x: hidden;
+	height: max-content;
```

Comment:
what's the purpose of this being in html?

Reply:
This at one point did solve an issue, however it appears to be no longer necessary and can be safely deleted